wooting-macro-backend/src/lib.rs

```diff
@@ -379,35 +397,60 @@ fn check_macro_execution_efficiently(
     output
 }
 
+#[derive(Debug, Clone, Default)]
+struct KeysPressed(Arc<RwLock<Vec<rdev::Key>>>);
+
+impl Drop for KeysPressed {
+    fn drop(&mut self) {
+        self.0.blocking_read().iter().for_each(|key| {
+            match simulate(&rdev::EventType::KeyRelease(*key)) {
+                Ok(()) => info!("Releasing pressed button {:?}", key),
+                Err(err) => error!("We could not send a drop key release.\n{}", err),
+            }
+        });
+    }
+}
+
```

Comment:
Not sure if this helps or is useful but I tried implementing a drop for a key pressed to properly release it. Should fix some bugs we had (if working as assumed)

Reply:
I'm not a massive fan of this, it would be preferred to have a more explicit releasing of keys when necessary

Reply:
I'm not sure it even works properly. Might remove it or try to tweak it (tho not quite sure how)

Reply:
This needs to be reviewed still it looks like

Reply:
Removed: [8cd5137](https://github.com/WootingKb/wooting-macros/pull/194/commits/8cd5137ee596a7111f59e888c253abcfcc502ceb)